subiquity/controllers/refresh.py

```diff
@@ -205,8 +205,7 @@ async def check_for_update(self):
             return CheckState.UNAVAILABLE
 
     async def start_update(self):
-        update_marker = os.path.join(self.app.state_dir, 'updating')
-        open(update_marker, 'w').close()
+        open(self.app.state_path('updating'), 'w').close()
```

Comment:
Not for this PR, but maybe we want a  StateFile with touch() implemented as a helper, very much like your set_content() with empty payload.

Reply:
Yes probably no harm in that.